ts/features/newWallet/components/WalletCardsContainer.tsx

```diff
@@ -39,15 +36,9 @@ const WalletCardsContainer = () => {
   const cards = useIOSelector(selectSortedWalletCards);
   const selectedCategory = useIOSelector(selectWalletCategoryFilter);
 
-  const stackCards = cards.length > 4;
```

Comment:
Are we sure about this? According to the Design specifications, wallet cards should only appear stacked if there are 5 or more.